index.html

```diff
@@ -26,8 +26,125 @@ <h1 class="detail one">I'm Mariane<br>Glad to see<br>you</h1>
           <li><a href="#"><img src="image/Vector-(1).png" alt="Email icon" class="space"></a></li>
           <li><a href="#"><img src="image/Vector-(2).png" alt="Social media" class="space"></a></li>
         </ul>
-      </section>
-  </div>
-</main>
+     </section>
+    </div>
+  
+    <div class="cards">
```

Comment:
- [ ] Since this div is supposed to be your main work container, I think it could be a nice idea if you use a semantic tag, in Html best practice, a semantic tag is more descriptive and professional than div, please change it into a `section` tag instead.